recipes/scanning-drift-corr/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "scanning_drift_corr" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 07aa7e271ab423644fc577c7655e6030ce7833c6c10558b5774149f7d91e3668
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - matplotlib
```

https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib

```suggestion
    - matplotlib-base
```